### PR TITLE
[fix][doc] update cluster failover code snippet

### PR DESCRIPTION
**Pull request by labuladong**
Fixes: https://github.com/apache/pulsar/issues/19073

### CODE REVIEW

**Review by tisonkun — COMMENTED** [review]
Thanks for your contribution @labuladong!

Perhaps you should also update docs for 2.11.x and 2.10.x?

**Comment by labuladong** [comment]
I think you're right, updated @tisonkun

**Review by tisonkun — APPROVED** [review]
LGTM. cc @momo-jun @Technoboy-

File: docs/client-libraries-java.md [hunk 1]
```diff
@@ -1234,8 +1248,8 @@ Parameter|Default value|Required?|Description
...
 `switchBackDelay`|N/A|Yes|The delay before the Pulsar client switches from the backup cluster to the primary cluster.<br /><br />Automatic failover switchover is controlled by a probe task: <br /> 1) After the Pulsar client switches from the primary cluster to the backup cluster, the probe task continues to check the status of the primary cluster. <br /> 2) If the primary cluster functions well and continuously remains active longer than `switchBackDelay`, the Pulsar client switches back to the primary cluster.
 `checkInterval`|30s|No|Frequency of performing a probe task (in seconds).
-`secondaryTlsTrustCertsFilePath`|N/A|No|Path to the trusted TLS certificate file of the backup cluster.
-`secondaryAuthentication`|N/A|No|Authentication of the backup cluster.
+`secondaryTlsTrustCertsFilePath`|N/A|No|A map of certificate file. Keys are service urls of backup cluster. Values are paths to the trusted TLS certificate file of the backup cluster.
```
Comment (momo-jun) [inline docs/client-libraries-java.md:1251]:
```suggestion
`secondaryTlsTrustCertsFilePath`|N/A|No|A map of certificate file. Keys are service URLs of the backup cluster. Values are paths to the trusted TLS certificate file of the backup cluster.
```

File: docs/client-libraries-java.md [hunk 2]
```diff
@@ -1234,8 +1248,8 @@ Parameter|Default value|Required?|Description
...
+`secondaryAuthentication`|N/A|No|A map of Authentication config. Keys are service urls of backup cluster. Values are Authentication object of the backup cluster.
```
Comment (momo-jun) [inline docs/client-libraries-java.md:1252]:
```suggestion
`secondaryAuthentication`|N/A|No|A map of Authentication config. Keys are service URLs of the backup cluster. Values are the authentication object of the backup cluster.
```

**Review by momo-jun — APPROVED** [review]
@labuladong Thanks for updating the docs. Only added a couple of suggestions.

**Review by Technoboy- — APPROVED** [review]

**Comment by labuladong** [comment]
Ready to merge

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**